gpos/xml/scap/ssg/content/ssg-chainguard-gpos-ds.xml

```diff
@@ -166,7 +167,10 @@
 
             </ns0:Profile>
             <ns0:Group id="xccdf_mil.disa.stig_group_V-263651">
-                <ns0:Rule id="xccdf_mil.disa.stig_rule_SV-263651r1_rule" selected="true" severity="medium">
+        <ns0:title>Security Group - xccdf_mil.disa.stig_group_V-263651</ns0:title>
+                <ns0:title xml:lang="en">SRG-OS Group V-263651</ns0:title>
+                <ns0:description xml:lang="en">&lt;GroupDescription&gt;&lt;/GroupDescription&gt;</ns0:description>
+                <ns0:Rule id="xccdf_mil.disa.stig_rule_SV-263651r1_rule" selected="true" severity="medium" weight="10.0">
```

Comment:
The rules (encoded as SV-NNNNNrREVISION) have a more complex revision number than "r1" in the DISA reference xccdf, e.g. for `V-263651`, the related SV rule id is `SV-263651r982555_rule`.

The added group description here doesn't seem useful, since it's not got any content.

Adding in the weight field I guess is fine, in the DISA STIG they all seem to be 10.0.

Reply:
Fixed in: [`35c8b42` (#20)](https://github.com/chainguard-dev/stigs/pull/20/commits/35c8b42d08dabf4f04c912a8a698ee9fd7a423d6)